src/common/utils/create_entity.py

```diff
@@ -69,21 +64,8 @@ def entity(self):
 
     @staticmethod
     def is_json(attr: BlueprintAttribute):
-        try:
-            json.loads(attr.default)
-            return True
-        except JSONDecodeError:
-            return False
-        except StopIteration:
-            return False
-
-    @staticmethod
-    def parse_json(attr: BlueprintAttribute):
-        try:
-            return json.loads(attr.default)
-        except JSONDecodeError:
-            print(f"invalid default value: {attr.default} for attribute: {attr}")
-            return ""
+        """A blueprint attribute is json if the default value is either a list or a dict."""
+        return type(attr.default) == dict or type(attr.default) == list
```

Comment:
Could be a bit more readable;
return attr.attributeType not in PRIMITIVE or attr.dimension != ""